tendrl/commons/objects/definition/master.yaml

```diff
@@ -439,4 +439,6 @@ namespace.tendrl:
       help: "Platform of the Node"
       value: nodes/$Node_context.node_id/Platform
       list: nodes/$Node_context.node_id/Platform
-tendrl_schema_version: 0.3
\ No newline at end of file
+tendrl_schema_version: 0.3
+min_reqd_gluster_ver: 3.9.0
```

Comment:
Put this inside a namespace man, how are we going to map this inside tendrl?

Please define these inside the "namespace.tendrl"